docs/setup/upgrade.md

```diff
@@ -2,42 +2,44 @@
 title: Upgrading KubeEdge
 sidebar_position: 8
 ---
-Please refer to following guide to upgrade your KubeEdge cluster.
+
+This guide provides instructions on how to upgrade your KubeEdge cluster.
 
 ## Backup
 
 ### Database
 
-Backup edgecore database at each edge node:
+Backup the EdgeCore database at each edge node:
 
 ```
 $ mkdir -p /tmp/kubeedge_backup
 $ cp /var/lib/kubeedge/edgecore.db /tmp/kubeedge_backup/
 ```
 
-### Config(Optional)
-
-You can keep old config to save some custom changes as you wish.
+### Config (Optional)
 
-*Note*:
+You can keep the old config to preserve any custom changes you have made.
 
-After upgrading, some options may be deleted and some may be added, please don't use old config directly.
+:::note
+After upgrading, some options may be removed, and new options may be added. It is not recommended to use the old config directly.
+:::
 
-### Device related(Optional)
+### Device related (Optional)
 
-If you upgrade from 1.3 to 1.4, please note that we upgrade device API from v1alpha1 to v1alpha2.
+If you are upgrading from KubeEdge v1.3 to v1.4, please note that device API should be upgraded from v1alpha1 to v1alpha2.
 
-You need to install [Device v1alpha2](https://github.com/kubeedge/kubeedge/blob/release-1.4/build/crds/devices/devices_v1alpha2_device.yaml)
+You will have to install [Device v1alpha2](https://github.com/kubeedge/kubeedge/blob/release-1.4/build/crds/devices/devices_v1alpha2_device.yaml)
 and [DeviceModel v1alpha2](https://github.com/kubeedge/kubeedge/blob/release-1.4/build/crds/devices/devices_v1alpha2_devicemodel.yaml),
-and manually convert their existing custom resources from v1alpha1 to v1alpha2.
+and manually convert your existing custom resources from v1alpha1 to v1alpha2.
 
-It's recommended to keep v1alpha1 CRD and custom resources in the cluster or exported somewhere, in case any rollback is needed.
+It is recommended to keep the v1alpha1 CRD and custom resources in the cluster or export them somewhere, in case a rollback is needed.
```

Comment:
```suggestion
It is recommended to keep the v1alpha2 CRD and custom resources in the cluster or export them somewhere, in case a rollback is needed.
```